build/vsts-ci.yml

```diff
@@ -2,309 +2,317 @@
-      publishUsingPipelines: true
-      pool:
-        name: NetCore1ESPool-Svc-Internal
-        demands: ImageOverride -equals windows.vs2019.amd64
-
-- template: /eng/common/templates/post-build/post-build.yml
-  parameters:
-    publishingInfraVersion: 3
-    # This is to enable SDL runs part of Post-Build Validation Stage
-    SDLValidationParameters:
-      enable: true
-      publishGdn: true
-      continueOnError: false
-      params: ' -SourceToolsList @("policheck","credscan")
-      -TsaInstanceURL https://devdiv.visualstudio.com/
-      -TsaProjectName DEVDIV
-      -TsaNotificationEmail mlnetcore@microsoft.com
-      -TsaCodebaseAdmin REDMOND\ericstj
-      -TsaBugAreaPath "DevDiv\Net Libraries"
-      -TsaIterationPath DevDiv
-      -TsaRepositoryName machinelearning
-      -TsaCodebaseName machinelearning
-      -TsaPublish $True'
\ No newline at end of file
+      jobs:
+      ################################################################################
+      - job: Linux_x64
+      ################################################################################
+        pool:
+          name: $(DncEngInternalBuildPool)
+          image: 1es-ubuntu-2204-pt
+          os: linux
+        container: CentosContainer
+        templateContext:
+          outputs:
+          - output: buildArtifacts
+            PathtoPublish: $(Build.SourcesDirectory)/artifacts/pkgassets
+            ArtifactName: pkgassets
+        steps:
+        # Only build native assets to avoid conflicts.
+        - script: ./build.sh -configuration $(BuildConfig) -projects $(Build.SourcesDirectory)/src/Native/Native.proj /p:TargetArchitecture=x64 /p:CopyPackageAssets=true
+          displayName: Build Native Assets
+
+      ################################################################################
+      - job: Linux_arm
+      ################################################################################
+        variables:
+          ROOTFS_DIR: '/crossrootfs/arm'
+        pool:
+          name: $(DncEngInternalBuildPool)
+          image: 1es-ubuntu-2204-pt
+          os: linux
+        container: UbuntuCrossArmContainer
+        templateContext:
+          outputs:
+          - output: buildArtifacts
+            PathtoPublish: $(Build.SourcesDirectory)/artifacts/pkgassets
+            ArtifactName: pkgassets
+        steps:
+        # Only build native assets to avoid conflicts.
+        - script: ./build.sh -configuration $(BuildConfig) -projects $(Build.SourcesDirectory)/src/Native/Native.proj /p:TargetArchitecture=arm /p:CopyPackageAssets=true
+          displayName: Build Native Assets
+
+      ################################################################################
+      - job: Linux_arm64
+      ################################################################################
+        variables:
+          ROOTFS_DIR: '/crossrootfs/arm64'
+        pool:
+          name: $(DncEngInternalBuildPool)
```

Comment:
Is there a way to deduplicate the name/image/OS pool details?

Reply:
`matrix:`, except that doesn't work in 1ES templates :upside_down_face: 

(Strictly speaking yes we could put all the specific parameters as custom properties of a top-level pool parameter object then foreach the object, but I didn't want to make major structural changes outside the scope of 1ES templates)

Reply:
I see, I was hoping for some sort of indirection based on the pool name.  Could we pull out `1es-ubuntu-2204-pt` into something like a variable name `$(LinuxHostImage)` or something like that?

Reply:
Oh, right, yes, I'll do that